### DISCUSSION TITLE
How to properly move completed torrents to an existing container?

### DISCUSSION THREAD

**Discussion by BalloonPark** [haugene/docker-transmission-openvpn#2312] [Q&A]
Previously I was able to move completed torrents to an install on a different system by just copying the contents of the \completed folder and the torrent files in transmission-home\torrents. 

I recently tried to do this again with a container that already had some torrents and I had been running for a while. I copied my completed files and torrents to the data and torrent folders. After doing this I restarted the container but can no logner access the web UI. The logs/portainer/unraid seem to indicate the container started and is working, but I don't see any CPU or network activity. When I try to access the webui, it does not load.

This container had been running for several months on an Unraid server. Let me know if any additional information is needed.

If this current container can't be salvaged, I would like to create a new container and migrate all my completed torrents there properly so this doesn't happen again.

I'm not very tech savvy and I appreciate any help!

My logs

> Starting container with revision: de2598df732811bfb4ac25395c5f27d6fa3b0d87
Creating TUN device /dev/net/tun
Using OpenVPN provider: PIA
Provider PIA has a custom setup script, executing it
Downloading OpenVPN config bundle openvpn into temporary file /tmp/tmp.OBIbkP
Extract OpenVPN config bundle into PIA directory /etc/openvpn/pia
Starting OpenVPN using config denmark.ovpn
Modifying /etc/openvpn/pia/denmark.ovpn for best behaviour in this container
Setting OpenVPN credentials...
adding route to local network 192.168.1.0/24 via 172.18.0.1 dev eth0
Fri Aug 12 03:52:11 2022 OpenVPN 2.4.9 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Apr 20 2020
Fri Aug 12 03:52:11 2022 library versions: OpenSSL 1.1.1g  21 Apr 2020, LZO 2.10
Fri Aug 12 03:52:11 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Fri Aug 12 03:52:11 2022 CRL: loaded 1 CRLs from file [[INLINE]]
Fri Aug 12 03:52:11 2022 TCP/UDP: Preserving recently used remote address: [AF_INET]46.246.106.36:1198
Fri Aug 12 03:52:11 2022 UDP link local: (not bound)
Fri Aug 12 03:52:11 2022 UDP link remote: [AF_INET]46.246.106.36:1198
Fri Aug 12 03:52:11 2022 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
Fri Aug 12 03:52:11 2022 [copenhagen408] Peer Connection Initiated with [AF_INET]46.246.106.36:1198
Fri Aug 12 03:52:12 2022 OpenVPN ROUTE6: OpenVPN needs a gateway parameter for a --route-ipv6 option and no default was specified by either --route-ipv6-gateway or --ifconfig-ipv6 options
Fri Aug 12 03:52:12 2022 OpenVPN ROUTE: failed to parse/resolve route for host/network: 2000::/3
Fri Aug 12 03:52:12 2022 TUN/TAP device tun0 opened
Fri Aug 12 03:52:12 2022 /sbin/ip link set dev tun0 up mtu 1500
Fri Aug 12 03:52:12 2022 /sbin/ip addr add dev tun0 10.2.112.107/24 broadcast 10.2.112.255
Fri Aug 12 03:52:12 2022 /etc/openvpn/tunnelUp.sh tun0 1500 1553 10.2.112.107 255.255.255.0 init
Up script executed with tun0 1500 1553 10.2.112.107 255.255.255.0 init
Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.2.112.107
Using Transmission Web Control  UI, overriding TRANSMISSION_WEB_HOME
Updating Transmission settings.json with values from env variables
Using existing settings.json for Transmission /data/transmission-home/settings.json
Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.2.112.107
Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /data/completed
Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /data/incomplete
Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091
Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /data/watch
sed'ing True to true
Transmission will run as
User name:   root
User uid:    0
User gid:    0
STARTING TRANSMISSION
Provider PIA has a script for automatic port forwarding. Will run it now.
If you want to disable this, set environment variable DISABLE_PORT_UPDATER=true
Transmission startup script complete.
Fri Aug 12 03:52:13 2022 WARNING: OpenVPN was configured to add an IPv6 route over tun0. However, no IPv6 has been configured for this interface, therefore the route installation may fail or may not work as expected.
Fri Aug 12 03:52:13 2022 Initialization Sequence Completed
port is 38488
the port has been bound to 38488  Fri Aug 12 03:52:20 UTC 2022
transmission auth not required
waiting for transmission to become responsive
transmission became responsive
   587     0%       None  Unknown      0.0     0.0   None  Idle         ***
Sum:            103.7 MB               0.0  38991.0
setting transmission port to 38488
localhost:9091/transmission/rpc/ responded: "success"
Checking port...
[2022-08-12 03:53:30.670] transmission-remote:  (http://localhost:9091/transmission/rpc/) Timeout was reached
initial setup complete!
waiting for rebind loop.................
token expiry 1665676283
remaining = 5399873
the port has been bound to 38488  Fri Aug 12 04:23:30 UTC 2022

My docker compose

> version: '2'
services:
    transmission-openvpn:
        volumes:
            - '/mnt/user/Volopt/Torrent:/data'
        environment:
            - OPENVPN_PROVIDER=PIA
            - OPENVPN_CONFIG=denmark
            - OPENVPN_USERNAME=***
            - OPENVPN_PASSWORD=***
            - WEBPROXY_ENABLED=false
            - LOCAL_NETWORK=192.168.1.0/24
            - PUID=0 
            - PGID=0
        cap_add:
            - NET_ADMIN
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - '9091:9091'
        image: haugene/transmission-openvpn:3.3

My variables

CREATE_TUN_DEVICE | true
-- | --
DROP_DEFAULT_ROUTE |  
ENABLE_UFW | false
GLOBAL_APPLY_PERMISSIONS | false
HEALTH_CHECK_HOST | google.com
LOCAL_NETWORK | 192.168.1.0/24
LOG_TO_STDOUT | false
OPENVPN_CONFIG | denmark
OPENVPN_PASSWORD | ***
OPENVPN_PROVIDER | PIA
OPENVPN_USERNAME | ***
PATH | /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PGID | 0
PUID | 0
REVISION | de2598df732811bfb4ac25395c5f27d6fa3b0d87
TRANSMISSION_DOWNLOAD_DIR | /data/completed
TRANSMISSION_HOME | /data/transmission-home
TRANSMISSION_INCOMPLETE_DIR | /data/incomplete
TRANSMISSION_RPC_PORT | 9091
TRANSMISSION_WATCH_DIR | /data/watch
TRANSMISSION_WEB_UI | transmission-web-control
UFW_ALLOW_GW_NET | false
UFW_DISABLE_IPTABLES_REJECT | false
UFW_EXTRA_PORTS |  
WEBPROXY_ENABLED | false
WEBPROXY_PASSWORD |  
WEBPROXY_PORT | 8888
WEBPROXY_USERNAME |

**Comment by pkishino** [comment]
You are using a very very old version of the container.
First try using :dev instead
